rust/backend/src/server.rs

```diff
@@ -0,0 +1,87 @@
+// SPDX-FileCopyrightText: 2024 Benedikt Zinn <benedikt.wh.zinn@gmail.com>
+//
+// SPDX-License-Identifier: MIT
+
+use crate::{configuration, constants};
+use configuration::Configuration;
+use shared::config::Configuration as ProtoConfig;
+use shared::ziofa::ziofa_server::{Ziofa, ZiofaServer};
+use shared::ziofa::{
+    CheckServerResponse,
+    // EbpfStreamObject,
+    Process, ProcessList, SetConfigurationResponse,
+};
+use tonic::{transport::Server, Request, Response, Status};
+
+#[derive(Default)]
+pub struct ZiofaImpl {
+    // tx: Option<Sender<Result<EbpfStreamObject, Status>>>,
+}
+
+#[tonic::async_trait]
+impl Ziofa for ZiofaImpl {
+    async fn check_server(&self, _: Request<()>) -> Result<Response<CheckServerResponse>, Status> {
+        // dummy data
+        let response = CheckServerResponse {};
+        Ok(Response::new(response))
+    }
```

Comment:
For healthchecks we can be compliant with the convention for grpc health checks, see https://grpc.io/docs/guides/health-checking/ and https://docs.rs/tonic-health/latest/tonic_health/

Reply:
Alright, no worries!